spec/Section 3 -- Type System.md

```diff
@@ -2164,3 +2173,115 @@ to the relevant IETF specification.
 ```graphql example
 scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
 ```
+
+### @defer
+
+```graphql
+directive @defer(
+  if: Boolean! = true
+  label: String
+) on FRAGMENT_SPREAD | INLINE_FRAGMENT
+```
+
+The `@defer` directive may be provided on a fragment spread or inline fragment
+to indicate that execution of the related selection set should be deferred. When
+a request includes the `@defer` directive, the result may consist of multiple
```

Comment:
When a request includes the `@defer` directive, it may return an _incremental stream_, consisting of an initial response containing all non-deferred data, followed by ... subsequent payloads including the deferred data.